src/Core/SettingsChangesHistory.h

```diff
@@ -89,6 +89,7 @@ static std::map<ClickHouseVersion, SettingsChangesHistory::SettingsChanges> sett
               {"ignore_drop_queries_probability", 0, 0, "Allow to ignore drop queries in server with specified probability for testing purposes"},
               {"lightweight_deletes_sync", 2, 2, "The same as 'mutation_sync', but controls only execution of lightweight deletes"},
               {"query_cache_system_table_handling", "save", "throw", "The query cache no longer caches results of queries against system tables"},
+              {"input_format_json_throw_on_duplicated_fields", false, true, "Throw an exception if there are duplicated fields in JSON object in JSON input formats. If disabled, the first field will be used"},
```

Comment:
```suggestion
              {"input_format_json_throw_on_duplicated_fields", true, true, "Throw an exception if there are duplicated fields in JSON object in JSON input formats. If disabled, the first field will be used"},
```

In my understanding previously we thrown an exception, so the previous value of this should `true`, not `false`.